tests/integration/views/test_static.py

```diff
@@ -0,0 +1,16 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestStatic(IntegrationTestCase):
+    def test_contact(self):
+        self.launchSurvey('test', '0205')
+        self.get('/contact-us')
+        self.assertInPage('Telephone')
+
+    def test_contact_with_no_metadata(self):
+        self.get('/contact-us')
+        self.assertInPage('Sorry, something has gone wrong.')
```

Comment:
this page shouldn't error should it?

Reply:
@jonnyshaw89 This should error as they would only see that page if they are an authenticated user i.e. logged in

Reply:
@MIslam92 i have just confirmed the behaviour with @charlotte-hirst.
When not logged in the behaviour should be that "two lines of text are not there "Please quote ..." and "Your survey id ..""
The card will be updated with this.